go.mod

```diff
@@ -108,8 +108,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.13.1
-	istio.io/api v1.19.0-alpha.1.0.20231025161828-8716c2d77491
-	istio.io/client-go v1.19.0-alpha.1.0.20231025162628-18b8989cc87a
+	istio.io/api v1.20.0-beta.0
```

Comment:
Not sure we want this, this is off the release-1.20 branch, we want head of `master` (or just don't change it, if its new enough already)

Reply:
Yup I know this part isn’t correct for how Istio vendors the API repo — I wasn’t sure on the exact process we used and didn’t see the process in my quick searching.

Reply:
Turns out it was recent enough already, I just didn't update my cache before a build 🙃 PR doesn't touch go.mod now.